docs/cases/api/cases-api/cases-api-find-cases-by-alert.asciidoc

```diff
@@ -1,33 +1,33 @@
 [[cases-api-find-cases-by-alert]]
-=== Find Cases by Alert
+=== Find cases by alert
 
-Retrieves all case IDs that have a specific alert attached to the cases.
+Uses an alert ID (`alert_id`) to retrieve case IDs of the cases the alert is attached to.
 
-NOTE: Console supports only Elasticsearch APIs. Console doesn't allow interactions with {kib} APIs. You must use `curl` or another HTTP tool instead. For more information, refer to {kibana-ref}/console-kibana.html[Run {es} API requests].
+TIP: The case ID is identical to the case's saved object ID.
```

Comment:
```suggestion
TIP: The case's case ID is identical to its saved object ID.
```

Reply:
@benironside what are your thoughts on shortening it to just "case ID" instead of using "case's case ID"?

Reply:
My $0.02: I lean toward "The case ID" -- shorter, less repetitive.

Reply:
+1 for shortening to `case ID`.